frontend/src/components/MyPaper.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
```

Comment:
This file and `Paper.svelte` is really similar. I do not like having both Components doing essentially the same thing. 

Maybe we can define a more generic Paper Component with all properties as optional parameters and only display the parameters which are not `null`?

What do you think?